### PR TITLE
Implement accept admin invitation screen

### LINKED ISSUE #518: [UI] Implement accept admin invitation screen
- [x] Create `AcceptPortalAdminInvitationScreen` component
  - [x] obtain invitation code from url
  - [x] display accept button
- [x] Handle redirection if not yet logged in
  - [x] redirect to login page
  - [x] back to invitation accept screen
- [x] Handle accept collaborator invitation mutation
- [x] Handle mutation error
- [x] Direct to app specific screen after successfully accept invitation

### PR DESCRIPTION
**Pull request by nicolaslee2775**
close #518 

##### Accept admin invitation screen
image
##### Invalid invitation code error
image

### CODE REVIEW

**Comment by axolotl0212** [comment]
#509 is merged, please rebase

File: portal/src/graphql/portal/AcceptAdminInvitationScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (appID !== null) {
+          navigate("../../");
+        }
+      })
+      .catch(() => {});
+  }, [acceptCollaboratorInvitation, invitationCode, navigate]);
+
+  return (
+    <main className={cn(styles.root, { [styles.loading]: loading })}>
+      {error && <ShowError error={error} />}
```
Comment (axolotl0212) [inline portal/src/graphql/portal/AcceptAdminInvitationScreen.tsx:39]: The error thrown by backend should contain a `reason` field? I think we should handle the error and show localized error message.

**Comment by louischan-oursky** [comment]
This screen is for accepting invitation to an unknown app so we should not see the side bar. In `pkg/portal/endpoint/endpoint`, the endpoint is `collaborators/invitation` so the path served by the portal client should be `/collaborators/invitation`.

**Comment by axolotl0212** [comment]
Actually I think this page should not be included in AppRouter (AppRoot.tsx) as this page is not `within` any app? The URL should look something like `localhost:8000/collaborators/invitation?code={base64 string}`. The appID should be encoded in the code so we should only include the appID into the url when user accepted the invitation and chosen the new app?

File: portal/src/AppRoot.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ const AppRoot: React.FC = function AppRoot() {
   }, [appID]);
   return (
     <ApolloProvider client={client}>
-      <ScreenLayout>
+      <ScreenLayout showScreenNav={true}>
```
Comment (louischan-oursky) [inline portal/src/AppRoot.tsx:36]: I think we should not be touching AppRoot and ScreenLayout. Can we follow the structure of `AppsScreen`?

**Review by axolotl0212 — APPROVED** [review]

**Comment by nicolaslee2775** [comment]
@louischan-oursky @axolotl0212 Updated.

**Review by louischan-oursky — APPROVED** [review]
Thanks

**Pull request merged**